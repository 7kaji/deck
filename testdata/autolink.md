# Autolink Test Cases

This is a normal link: [example](https://example.com)

Bracketed autolinks (should be converted):
- <https://example.com>

Naked URLs (should NOT be converted):
- https://example.com

Mixed content:
- Visit <https://github.com> for more info
- Or go to https://github.com directly
