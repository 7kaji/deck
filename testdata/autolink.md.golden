[
  {
    "layout": "title-and-body",
    "titles": [
      "Autolink Test Cases"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "This is a normal link: "
              },
              {
                "value": "example",
                "link": "https://example.com"
              }
            ]
          },
          {
            "fragments": [
              {
                "value": "Bracketed autolinks (should be converted):"
              }
            ]
          },
          {
            "fragments": [
              {
                "value": "https://example.com",
                "link": "https://example.com"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "Naked URLs (should NOT be converted):"
              }
            ]
          },
          {
            "fragments": [
              {
                "value": "https://example.com"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "Mixed content:"
              }
            ]
          },
          {
            "fragments": [
              {
                "value": "Visit "
              },
              {
                "value": "https://github.com",
                "link": "https://github.com"
              },
              {
                "value": " for more info"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "Or go to https://github.com directly"
              }
            ],
            "bullet": "-"
          }
        ]
      }
    ],
    "title_heading_level": 1
  }
]